modules/nf-core/vcftools/main.nf

```diff
@@ -91,10 +91,14 @@ process VCFTOOLS {
 
     def bed_arg  = (args.contains('--bed')) ? "--bed ${bed}" :
         (args.contains('--exclude-bed')) ? "--exclude-bed ${bed}" :
-        (args.contains('--hapcount')) ? "--hapcount ${bed}" : ''
+        (args.contains('--hapcount')) ? "--hapcount ${bed}" :
+        (args.contains('--positions')) ? "--positions ${bed}" :
+        (args.contains('--exclude-positions')) ? "--exclude-positions ${bed}"  : ''
```

Comment:
Are the different args exclusive?

Reply:
Yes, I think so.

Reply:
@maxulysse Hi, Maxime. By any chance you can approve this and merge it? Thanks.